model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -98,6 +99,8 @@ data class VcsMatcher(
     }
 
     fun matches(vcsInfo: VcsInfo): Boolean =
-        type == vcsInfo.type && url == vcsInfo.url && (path == null || path == vcsInfo.path) &&
+        type == vcsInfo.type && matchUrls(url, vcsInfo.url) && (path == null || path == vcsInfo.path) &&
                 revision == vcsInfo.resolvedRevision
 }
+
+private fun matchUrls(lhs: String, rhs: String) = lhs.stripCredentialsFromUrl() == rhs.stripCredentialsFromUrl()
```

Comment:
The function name sounds like it would do something with the URLs, but not return a boolean. How about `matchesWithoutCredentials`, even if long?

Reply:
Done.